torch/csrc/jit/script/init.cpp

```diff
@@ -619,23 +624,31 @@ FunctionSchema getSchemaWithNameAndDefaults(
 void initJitScriptBindings(PyObject* module) {
   auto m = py::handle(module).cast<py::module>();
 
+  py::bind_map<ExtraFilesMap>(m, "ExtraFilesMap");
```

Comment:
so we need to do this because containers are not mutable by default? maybe add a comment?